kratos/python_scripts/restart_utility.py

```diff
@@ -213,3 +288,40 @@ def __GetPrettyTime(self, time):
         pretty_time = "{0:.12g}".format(time)
         pretty_time = float(pretty_time)
         return pretty_time
+
+    def __IsRestartFile(self, dir_entry):
```

Comment:
I recommend checkicg [`pathlib.Path`](https://docs.python.org/3/library/pathlib.html) which should make this easier / more readable